www-src/app/replicator/change/change_dispatcher.coffee

```diff
@@ -37,8 +36,9 @@ module.exports = class ChangeDispatcher
     dispatch: (doc, callback = ->) ->
         log.info "dispatch #{doc.docType}"
 
-        if @isDispatched(doc) and @changeHandlers[doc.docType.toLowerCase()]
```

Comment:
No worries.
This file is almost brand new and contains already 'historical' method, and trace of fight between devs : )